packages/app-project/src/screens/ProjectHomePage/components/Hero/components/Background/Background.js

```diff
@@ -1,45 +1,30 @@
-import { number, shape, string } from 'prop-types'
-import styled, { css, withTheme } from 'styled-components'
+import { string } from 'prop-types'
+import styled from 'styled-components'
 
 const Img = styled.img`
-    height: 230px;
-    min-height: inherit;
-    object-fit: cover;
-    width: 100%;
+  height: 230px;
+  min-height: inherit;
+  object-fit: cover;
+  width: 100%;
 
-  ${props => css`
-    @media (min-width: ${props.breakpoint}px) {
-      flex: 1 1 auto;
-      object-position: 0 50%;
-    }
-  `}
+  // Grommet small breakpoint
+  @media (width > 769px) {
+    flex: 1 1 auto;
+    object-position: 0 50%;
+  }
 `
```

Comment:
Passing props like `breakpoint` throws a warning in v6 because [shouldForwardProp is no longer the default](https://styled-components.com/docs/faqs#shouldforwardprop-is-no-longer-provided-by-default). I refactored this Background component to simply use CSS breakpoints of a defined pixel unit because that's what we do in other places like lib-content.